Dockerfile

```diff
@@ -0,0 +1,6 @@
+# see https://pptr.dev/guides/docker and https://github.com/puppeteer/puppeteer/pkgs/container/puppeteer
+FROM ghcr.io/puppeteer/puppeteer:23
+RUN npm install args-and-flags content-disposition fs-extra imapflow mailparser md5 nodemailer winston
+COPY --chown=node:node *.js .
+EXPOSE 1984
```

Comment:
Maybe mention that this is the totally arbitrary port of the status server?

Reply:
thx, added a comment